index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="navbar">
+        <img class="pic1" src="image_1.png" alt="geometry">
+        <img class="menu" src="ic_menu.png" alt="Menu">
+    </div>
```

Comment:
- [ ] Kindly use `header` tag instead of `div` here `<div class="navbar">`, because this this your header section.
- [ ] Kindly wrap your logo with `a` to link to your home page, every logo should link to the home page. Like `<a href='index.html'> <img class="pic1" src="image_1.png" alt="geometry"> </a>`.
- [ ] Kindly wrap your menu image with `nav`, like `<nav><img class="menu" src="ic_menu.png" alt="Menu"></nav>` , because you will navigate from this icon.

Reply:
resolved 👍